src/chaos/k8s_interaction_test.go

```diff
@@ -0,0 +1,97 @@
+package chaos
+
+import (
+	"context"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic/fake"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+func TestCheckUnhealthyNodeValidation(t *testing.T) {
```

Comment:
I tried to run go test, but I got an error message

```
--- FAIL: TestCheckUnhealthyNodeValidation (0.01s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xa0c76a]
```

Reply:
Somehow it works on my Ubuntu 22.04, go version go1.22.0 linux/amd64. Will need a closer look

Reply:
Figured it out. This happened because there was no ~/.kube/config file on your machine. I've changed logic so there is no error now